release-engineering/role-handbooks/branch-manager.md

```diff
@@ -73,38 +72,45 @@ In this handbook, we will make several references to Kubernetes releases, milest
 
 For the purposes of this handbook, we'll assume that:
 
-- the current release in development is Kubernetes 1.17
-- the previous release is Kubernetes 1.16
-- the next release is Kubernetes 1.18
-- the release no longer in support is Kubernetes 1.13
+- the current release in development is Kubernetes 1.18
+- the previous release is Kubernetes 1.17
+- the next release is Kubernetes 1.19
+- the release no longer in support is Kubernetes 1.14
 
 To simplify certain instructions, we will make the following connections:
 
 | Text | SemVer | Reference Release |
 |---|---|---|
-| "current release", "current milestone", "in development" | `x.y` | Kubernetes 1.17 |
-| "previous release", "previous milestone" | `x.y-1` | Kubernetes 1.16 |
-| "next release", "next milestone" | `x.y+1` | Kubernetes 1.18 |
-| "release no longer in support" | `x.y-4` | Kubernetes 1.13 |
+| "current release", "current milestone", "in development" | `x.y` | Kubernetes 1.18 |
+| "previous release", "previous milestone" | `x.y-1` | Kubernetes 1.17 |
+| "next release", "next milestone" | `x.y+1` | Kubernetes 1.19 |
+| "release no longer in support" | `x.y-4` | Kubernetes 1.14 |
 
 **As an editor of this content, Branch Managers should periodically update these conventions and the examples contained within this handbook.**
 
+### Minimum Requirements
+
+- Familiarity with basic Unix commands and able to debug shell scripts.
+- Familiarity with branched source code workflows via `git` and associated `git` command line invocations.
+- General knowledge of Google Cloud (Cloud Build and Cloud Storage).
```

Comment:
Could we lower the bar a bit by being more generic?  Eg:

- Abstract knowledge of public cloud provider patterns (eg: Amazon, Google, Microsoft, VMware, etc.), building code from containerize build environments, and cloud storage, with the ability to explore and map that knowledge to specifics on GCP (Cloud Build and Cloud Storage).

I think a lot of us didn't have for example GCB specific knowledge until we got into the role.  We should rather have specializing generalists who are able to work in the abstracted problem space versus people who can or will only solve a problem if it is being done on one specific provider.